sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -56,41 +56,9 @@ await using var client = new ServiceBusClient(connectionString);
 
 To see how to authenticate using Azure.Identity, view this [example](#authenticating-with-azureidentity).
 
-To see how to initiate the connection with a custom endpoint, view this [sample](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample13_AdvancedConfiguration.md#initiating-the-connection-with-a-custom-endpoint).
-
-### ASP.NET Core
-
-To inject `ServiceBusClient` as a dependency in an ASP.NET Core app, install the Azure client library integration for ASP.NET Core package.
-
-```dotnetcli
-dotnet add package Microsoft.Extensions.Azure
-```
-
-Then register the client in the `Startup.ConfigureServices` method:
-
-```csharp
-public void ConfigureServices(IServiceCollection services)
-{
-    services.AddAzureClients(builder =>
-    {
-        builder.AddServiceBusClient(Configuration.GetConnectionString("ServiceBus"));
-    });
-  
-    services.AddControllers();
-}
-```
-
-To use the preceding code, add this to your configuration:
-
-```json
-{
-  "ConnectionStrings": {
-    "ServiceBus": "<connection_string>"
-  }
-}
-```
+For examples of how to authenticate for an ASP.NET Core application, view this [example](#registering-with-aspnet-core-dependency-injection).
```

Comment:
@JoshLove-msft :  The "how do I register for DI with a credential" came up twice this week in support issues.   When I went to add the snippet, it felt large and like it didn't quite fit into "authorize the client".   

I tried moving it down to an example scenario and leaving a note here.   If you don't like it, I can move things back.   Thoughts?